src/CreditCard.js

```diff
@@ -0,0 +1,34 @@
+import './CreditCard.css'
+
+const CreditCard = (props) => {
+
+    let cardType
+
+    props.type === "Visa" ? cardType = "img/visa.png" : cardType = "img/master-card.svg"
+
+    let background = {
+        backgroundColor: `${props.bgColor}`,
+        color: `${props.color}`
+    }
+
+    let numbers = (props.number).substr(-4)
+
+
+
+    return (
+
+        <div style={background}>
+            <img src={cardType} alt="props.cardType" />
+            <p>● ● ● ● ● ● ● ● ● ● ● ● {numbers}</p>
+            <p>{props.expirationMonth}</p>
```

Comment:
Según vayáis cogiendo soltura con React este tipo de código que puede ir en una linea lo irás metiendo dentro del JSX con más soltura, te dejo una muestra.
```suggestion
const CreditCard = (props) => {

    return (
        <div style={{backgroundColor: `${props.bgColor}`,color: `${props.color}`}}>
            <img src={props.type === "Visa" ? "img/visa.png" :  "img/master-card.svg"} alt="props.cardType" />
            <p>● ● ● ● ● ● ● ● ● ● ● ● {(props.number).substr(-4)}</p>
            <p>{props.expirationMonth}</p>
```